android/libraries/rib-base/src/main/java/com/uber/rib/util/RIBs.java

```diff
@@ -84,16 +84,16 @@ static Configuration getConfiguration() {
      * Called when there is a message that should be logged for debugging. Consumers should route
      * this data to a debug logging location.
      *
-     * <p>If no configuration is set, the default implementation of this will drop the messages.
+     * <p>If no errorHandler is set, the default implementation of this will drop the messages.
      *
      * @param format Message format - See {@link String#format(String, Object...)}
      * @param args Arguments to use for printing the message.
      */
     void handleDebugMessage(String format, @Nullable final Object... args);
   }
 
-  /** Default, internal implementation that is used when host app does not set a configuration. */
-  private static class DefaultConfiguration implements Configuration {
+  /** Default, internal implementation that is used when host app does not set a errorHandler. */
+  private static class DefaultErrorHandler implements ErrorHandler {
```

Comment:
To be honest, we already use timber internally for these purposes
Do we still need to keep this class?